Ontology/Capability/Actuator/BinaryActuator/StartStopActuator/StartStopActuator.json

```diff
@@ -1,5 +1,5 @@
 {
-  "@id": "dtmi:digitaltwins:rec_3_3:device:RunStopActuator;1",
+  "@id": "dtmi:digitaltwins:rec_3_3:device:StartStopActuator;1",
   "@type": "Interface",
   "displayName": {
     "en": "Run/stop actuator"
```

Comment:
Do you want to add the display name to match Start/Stop actuator?

Reply:
Good catch, will fix and commit.